### PR TITLE
Show user account on grant loginflow step

### PR DESCRIPTION
**Pull request by nickvergessen**
## Login flow v2 (Desktop client)
Bildschirmfoto vom 2022-05-13 10-51-16

## Login flow v1 (Android app)
Screenshot_2022-05-13-10-46-16-298_com nextcloud talk2

### CODE REVIEW

File: core/templates/loginflow/grant.php [hunk 1]
```diff
@@ -44,7 +50,7 @@
 			<input type="hidden" name="requesttoken" value="<?php p($_['requesttoken']) ?>" />
 			<input type="hidden" name="stateToken" value="<?php p($_['stateToken']) ?>" />
 			<input type="hidden" name="oauthState" value="<?php p($_['oauthState']) ?>" />
-			<?php if (p($_['direct'])) { ?>
+			<?php if ($_['direct']) { ?>
```
Comment (nickvergessen) [inline core/templates/loginflow/grant.php:53]: This printed the white `0` on the screen of the flow :D 
`p()` is the printing function and the return of it is void, so the direct was first of all always lost and second the 0/1 wsa printed on the screen.

**Review by CarlSchwan — APPROVED** [review]

File: core/templates/loginflow/grant.php (hunk 1 above)
Comment (CarlSchwan) [reply core/templates/loginflow/grant.php:53]: yeah @skjnldsv pointed it out a few minutes ago https://github.com/nextcloud/server/pull/32374

Comment (skjnldsv) [reply core/templates/loginflow/grant.php:53]: @nickvergessen pointed it out to me :stuck_out_tongue:

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**